it/controllers/updateRegisteredDetails/BusinessAddressControllerISpec.scala

```diff
@@ -0,0 +1,36 @@
+package controllers.updateRegisteredDetails
+
+import controllers.ControllerITTestHelper
+import org.jsoup.Jsoup
+import org.scalatest.matchers.must.Matchers.{convertToAnyMustWrapper, include}
+import play.api.i18n.Messages
+import play.api.test.WsTestClient
+import models.SelectChange.UpdateRegisteredDetails
+
+class BusinessAddressControllerISpec extends ControllerITTestHelper {
+
+  val normalRoutePath = "/business-address"
+
+  "GET " + normalRoutePath - {
```

Comment:
I know they'd be very similar, but might be worth adding these for 'POST'

Reply:
The post doesn't do anything except redirect (which can be added during navigation ticket if needed), there is nothing needed to check in the ispec file - the post is handled within the controller spec.

Reply:
We wouldn't necessarily say it was one or the other in other cases though? It's just because this is an exceedingly simple case